.github/workflows/example-install-only.yml

```
@@ -32,5 +35,37 @@ jobs:
       - name: Cypress tests 🧪
         uses: ./
         with:
-          working-directory: examples/install-only
+          working-directory: examples/v9/install-only
+          install: false
+
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ Cypress v10 and higher ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ #
+
+  # do not install every dependency in this example
+  # just install Cypress, but make sure to cache it
+  install-cypress-only-v10:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout 🛎
+        uses: actions/checkout@v2
+
+      # cache NPM modules and Cypress binary folder
+      # we can use "package-lock.json" as the key file
+      # to make sure we use the precise Cypress version
+      # (which is important when using ^ version in package.json)
+      # see https://github.com/actions/cache
+      - name: Cache NPM and Cypress 📦
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.cache/Cypress
+            node_modules
+          key: my-cache-${{ runner.os }}-${{ hashFiles('package-lock.json') }}
+
+      - name: Install Cypress 📥
+        run: npm i https://cdn.cypress.io/beta/npm/10.0.0/linux-x64/circle-10.0-release-faa8fb5302ae8fac4ac53f1dcd214abe35bce3ab/cypress.tgz
```

Making a note that this will need to be updated once v10 is officially released. Would it be better to do this assuming this PR is merged after v10 release:

```suggestion
        run: npm i cypress
```

---

yes, we will need to update all of the Cypress binaries to `10.0.0` once it has been officially released and then make sure all of these examples and tests still work.